src/envs/behavior_options.py

```diff
@@ -461,46 +462,160 @@ def create_grasp_policy(
 
     def graspObjectOptionPolicy(_state: State,
                                 env: "BehaviorEnv") -> Tuple[Array, bool]:
+        nonlocal plan
+        nonlocal reversed_plan
```

Comment:
can we keep track of just `plan` or `reversed_plan` rather than both?

Reply:
it's possible but would require the use of a pointer instead of `pop()` like we're currently doing. My sense is that this will complicate the logic here a bit more, but tbh it's quite messy as is. I'll try to see if I can refactor to make it cleaner/easier to understand!

Reply:
when I wrote this I didn't understand that the policy first does the plan, then does the reversed_plan, so I think it's fine to keep both. thanks for clearing it up!